opencti-platform/opencti-front/src/private/components/settings/status_templates/StatusTemplates.tsx

```diff
@@ -1,118 +1,173 @@
+  )
+  @refetchable(queryName: "StatusTemplatesLinesRefetchQuery") {
+    statusTemplates(
+      search: $search
+      first: $count
+      after: $cursor
+      orderBy: $orderBy
+      orderMode: $orderMode
+    ) @connection(key: "Pagination_statusTemplates") {
+      edges {
+        node {
+          ...StatusTemplatesLine_node
+        }
+      }
+      pageInfo {
+        endCursor
+        hasNextPage
+        globalCount
+      }
+    }
+  }
+`;
+
+export const StatusTemplatesLineFragment = graphql`
+  fragment StatusTemplatesLine_node on StatusTemplate {
+    id
+    name
+    color
+    usages
+  }
+`;
 
 const LOCAL_STORAGE_KEY = 'status-templates';
 
 const StatusTemplates = () => {
   const { t_i18n } = useFormatter();
   const { setTitle } = useConnectedDocumentModifier();
   setTitle(t_i18n('Status Templates | Taxonomies | Settings'));
-  const classes = useStyles();
-  const { viewStorage, helpers, paginationOptions } = usePaginationLocalStorage<StatusTemplatesLinesPaginationQuery$variables>(
+
+  const initialValues = {
+    searchTerm: '',
+    sortBy: 'name',
+    orderAsc: false,
+    filters: emptyFilterGroup,
+  };
+
+  const {
+    viewStorage,
+    helpers,
+    paginationOptions,
+  } = usePaginationLocalStorage<StatusTemplatesLinesPaginationQuery$variables>(
     LOCAL_STORAGE_KEY,
-    {
-      searchTerm: '',
-      sortBy: 'name',
-      orderAsc: false,
-    },
+    initialValues,
   );
-  const renderLines = () => {
-    const { sortBy, orderAsc, searchTerm } = viewStorage;
-    const dataColumns = {
-      name: {
-        label: 'Name',
-        width: '50%',
-        isSortable: true,
-      },
-      color: {
-        label: 'Color',
-        width: '30%',
-        isSortable: false,
-      },
-      usages: {
-        label: 'Usages',
-        width: '20%',
-        isSortable: false,
-      },
-    };
-    const queryRef = useQueryLoading<StatusTemplatesLinesPaginationQuery>(
-      statusTemplatesLinesQuery,
-      paginationOptions,
-    );
-    return (
-      <ListLines
-        helpers={helpers}
-        sortBy={sortBy}
-        orderAsc={orderAsc}
-        dataColumns={dataColumns}
-        handleSort={helpers.handleSort}
-        handleSearch={helpers.handleSearch}
-        displayImport={false}
-        secondaryAction={false}
-        keyword={searchTerm}
-        createButton={
-          <StatusTemplateCreation
+
+  const { filters } = viewStorage;
+  const contextFilters = useBuildEntityTypeBasedFilterContext('StatusTemplates', filters);
+
+  const queryPaginationOptions = {
+    ...paginationOptions,
+    filters: contextFilters,
+  } as unknown as StatusTemplatesLinesPaginationQuery$variables;
+
+  const queryRef = useQueryLoading<StatusTemplatesLinesPaginationQuery>(
+    statusTemplatesQuery,
+    queryPaginationOptions,
+  );
+
+  const dataColumns: DataTableProps['dataColumns'] = {
+    name: {
+      id: 'name',
+      label: 'Name',
```

Comment:
adding precisions like label and isSortable is useless here since name is already declared in dataTable utils and don't need to be overwritten in this case - same for usages and color